contrib/ci/build_and_install_pkgs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+# prepare the build tree
+rm -rf build
+mkdir build && pushd build
+cp ../contrib/PKGBUILD .
+mkdir -p src/fwupd && pushd src/fwupd
+ln -s ../../../* .
+popd
+
+# build the package and install it
+makepkg -sei --noconfirm
+
+# move the package to working dir
+cp *.pkg.tar.xz ../
```

Comment:
Can you please leave a comment at the end of this script about why gnome-desktop-testing isn't currently activated?  Also, if possible file a bug with the Arch gnome maintainer and link to the bug number in the comment.